chromium_src/crypto/symmetric_key.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../crypto/symmetric_key.cc"
+
+namespace crypto {
+
+// static
+std::unique_ptr<SymmetricKey>
```

Comment:
this stuff all needs icwyu

Reply:
fixed in 217729570317fe5e22c165799eb78305138e7a0a